packages/server/database/migrations/20200803121128-addTemplateDimension.ts

```diff
@@ -0,0 +1,22 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function(r: R) {
+  try {
+    await r.tableCreate('TemplateDimension').run()
+
+    await r
+      .table('TemplateDimension')
+      .indexCreate('templateId')
+      .run()
+  } catch (e) {
+    console.log(e)
+  }
+}
+
+export const down = async function(r: R) {
+  try {
+    await r.tableDrop('TemplateDimension').run()
```

Comment:
i'm betting we'll need a `scaleId` for the foreign key, but safe to leave off until we need it!